docs/integrations/sources/shopify.md

```diff
@@ -133,15 +133,15 @@ This source can sync data for the [Shopify REST API](https://shopify.dev/api/adm
 - [Inventory Levels (GraphQL)](https://shopify.dev/docs/api/admin-graphql/2024-04/objects/InventoryLevel)
 - [Locations](https://shopify.dev/api/admin-rest/2024-04/resources/location)
 - [Metafields (GraphQL)](https://shopify.dev/docs/api/admin-graphql/2024-04/objects/Metafield)
+- [Order Agreements (GraphQL)](https://shopify.dev/docs/api/admin-graphql/2024-04/objects/OrderAgreement)
 - [Orders](https://shopify.dev/api/admin-rest/2024-04/resources/order#top)
 - [Order Refunds](https://shopify.dev/api/admin-rest/2024-04/resources/refund#top)
-- [Order Risks](https://shopify.dev/api/admin-rest/2024-04/resources/order-risk#top)
+- [Order Risks (GraphQL)](https://shopify.dev/api/admin-rest/2024-04/resources/order-risk#top)
 - [Pages](https://shopify.dev/api/admin-rest/2024-04/resources/page#top)
 - [Price Rules](https://shopify.dev/api/admin-rest/2024-04/resources/pricerule#top)
-- [Products](https://shopify.dev/api/admin-rest/2024-04/resources/product#top)
-- [Products (GraphQL)](https://shopify.dev/api/admin-graphql/2022-10/queries/products)
-- [Product Images](https://shopify.dev/api/admin-rest/2024-04/resources/product-image)
-- [Product Variants](https://shopify.dev/api/admin-rest/2024-04/resources/product-variant)
+- [Products (GraphQL)](https://shopify.dev/docs/api/admin-graphql/2024-04/queries/products)
+- [Product Images (GraphQL)](https://shopify.dev/docs/api/admin-graphql/2024-04/objects/Image)
+- [Product Variants (GraphQL)](https://shopify.dev/docs/api/admin-graphql/2024-04/queries/productVariant)
```

Comment:
Updated bunch of links to use the latest API version and check they are available, because this wasn't covered for the last couple of PRs.